threadit-web/src/containers/Sidebar/Sidebar.tsx

```diff
@@ -22,9 +32,9 @@ export const Sidebar = observer(() => {
             <Divider />
             <NavLink to={"/"}><Button leftIcon={<Icon as={IoStatsChart} />} colorScheme={"purple"}>Home</Button></NavLink>
```

Comment:
Should move the "Create Spool" button into the first section with "Home". 
<img width="277" alt="image" src="https://user-images.githubusercontent.com/82107652/219880380-83cf5e1f-8141-43b1-a71a-853e81aa1619.png">

Reply:
Also the "Create Spool" button should only be visible when logged in.